Reemit.Decompiler.Clr/Disassembler/OpcodeInfoComparer.cs

```diff
@@ -0,0 +1,20 @@
+﻿using System.Diagnostics.CodeAnalysis;
+
+namespace Reemit.Decompiler.Clr.Disassembler;
+
+public class OpcodeInfoComparer : IEqualityComparer<OpcodeInfo>
+{
+    public static OpcodeInfoComparer Instance = new OpcodeInfoComparer();
+
+    public bool Equals(OpcodeInfo? x, OpcodeInfo? y) =>
+        x?.Opcode == y?.Opcode && x?.ExtendedOpcode == y?.ExtendedOpcode;
+
+    public int GetHashCode([DisallowNull] OpcodeInfo obj)
```

Comment:
Looks like this one is redundant:
<img width="519" alt="image" src="https://github.com/user-attachments/assets/e4ad8ff1-5d87-4e5f-b41b-6c5d54a749f5">

Reply:
Done